themes/bootstrap3/templates/RecordTab/collectionlist.phtml

```diff
@@ -13,10 +13,17 @@
   <? endforeach; ?>
   <div class="clearfix hidden-print">
     <div class="pull-left flip">
-      <?=$this->transEsc("Showing")?>
-      <strong><?=$this->localizedNumber($results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($results->getEndRecord())?></strong>
+      <?
+        $transParams = [
```

Comment:
Do you have a test instance using collections that we can verify this template with? I don't currently have an easy way to test this one.